### PR TITLE
RPET-703: Adding cost details explaination

### PR DESCRIPTION
**Pull request by jakub-kozlowski**
# Description

- added `claimsCostExplain` to prayer/claim-costs template and JS file

Fixes https://tools.hmcts.net/jira/browse/RPET-703

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/steps/prayer/claim-costs/content.json [hunk 1]
```diff
@@ -45,6 +46,7 @@
           "warning5YearsSeparation": "Fel rheol ni fydd y llys yn gwneud gorchymyn costau pan fo’r cais yn seiliedig ar fyw ar wahân ers 5 mlynedd.",
           "whoCosts": "Gan bwy ydych chi am hawlio costau?",
           "your": "Eich",
+          "costDetails": "Manylion cost",
```
Comment (Dizrupto) [inline app/steps/prayer/claim-costs/content.json:49]: nice, do you speak welsh? :D

**Review by Dizrupto — APPROVED** [review]

File: app/steps/prayer/claim-costs/content.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
           "warning5YearsSeparation": "The court does not normally make a costs order where the application is based on 5 years separation.",
           "whoCosts": "Who do you want to claim the costs from?",
           "your": "Your",
+          "costDetails": "Cost details",
           "theCorespondent": "{{ reasonForDivorceAdultery3rdPartyFirstName }} {{ reasonForDivorceAdultery3rdPartyLastName }} (the co-respondent)",
           "youHaveToPayFeeNow": "The court will make a judgement on who should pay, and how much, at a later stage in the divorce.",
           "youHaveToPayFeeAppNow": "You’ll have to pay your application fee now. ",
```
Comment (madjava) [inline app/steps/prayer/claim-costs/content.json:17]: Could we just call this `youHaveToPayFee` and `youHaveToPayFeeApplication`? The urgency is not needed in the code 😜

**Review by madjava — APPROVED** [review]

File: app/steps/prayer/claim-costs/content.json (hunk 2 above)
Comment (jakub-kozlowski) [reply app/steps/prayer/claim-costs/content.json:17]: @madjava - this is not my change, mine is just the one-liner adding `costDetails`

File: app/steps/prayer/claim-costs/content.json (hunk 1 above)
Comment (jakub-kozlowski) [reply app/steps/prayer/claim-costs/content.json:49]: Not at all - Google translate... Asked Welsh friend but he said his Welsh is not that good actually, but confirmed manylion is cost :-)

**Comment by jakub-kozlowski** [comment]
https://tools.hmcts.net/jira/browse/RPET-703 - AC changed to remove cost details explanation from PFE - closing and will delete branch once ticket is done.

**Pull request closed without merging**